src/AppBundle/Service/WebsiteCommunicationService.php

```diff
@@ -0,0 +1,176 @@
+          ]
+      );
+
+      $response = $client->request(
+        'POST',
+        $this->endpoint . "/api/udstykning/" . $salgsomraade->getId() . "/grunde",
+        [
+          'auth' => [$this->username, $this->password],
+          'json' => $this->getSalgsomraadePublicFields($salgsomraade)
+        ]
+      );
+
+      $response = $client->request(
+        'POST',
+        $this->endpoint . "/api/udstykning/" . $salgsomraade->getId() . "/grunde/geojson",
+        [
+          'auth' => [$this->username, $this->password],
+          'json' => $this->getSalgsomraadePublicFields($salgsomraade)
+        ]
+      );
+
+      $body = $response->getBody()->getContents();
+      $content = \GuzzleHttp\json_decode($body);
+
+      return (array) $content;
+    } else {
+      return FALSE;
+    }
+  }
+
+  public function getSalgsomraadePublicFields(Salgsomraade $salgsomraade): array {
+    return [
+      'id' => $salgsomraade->getId(),
+      'type' => $salgsomraade->getType(),
+      'title' => $salgsomraade->getTitel(),
+      'vej' => $salgsomraade->getVej(),
+      'city' => $salgsomraade->getPostby() ? $salgsomraade->getPostby()->getCity() : null,
+      'postalCode' => $salgsomraade->getPostby() ? $salgsomraade->getPostby()->getPostalcode() : null,
+      'geometry' => $salgsomraade->getSpGeometryArray(),
+      'srid' => $salgsomraade->getSrid(),
+      'publish' => $salgsomraade->isAnnonceres(),
+    ];
+  }
+
+  public function getGrundeGeoJsonFields(int $salgsomraadeId): array {
+    $grunde = $this->entityManager->getRepository('AppBundle:Grund')->getGrundeForSalgsOmraade($salgsomraadeId);
+
+    $list = [];
+    $list['type'] = 'FeatureCollection';
+
+    $crs                       = [];
+    $crs['type']               = 'link';
+    $crs['properties']['href'] = 'http://spatialreference.org/ref/epsg/25832/proj4/';
+    $crs['properties']['href'] = $this->getParameter('crs_properties_href');
+    $crs['properties']['type'] = $this->getParameter('crs_properties_type');
+    $list['crs']               = $crs;
+
+    $list['features'] = [];
+
+    foreach ($grunde as $grund) {
+      $data = [];
+
+      $data['type']     = 'Feature';
+      $data['geometry'] = $grund->getSpGeometryArray();
+
+      $properties['id']      = $grund->getId();
+      $properties['address'] = trim($grund->getVej().' '.$grund->getHusnummer().$grund->getBogstav());
+      $properties['status']  = $this->publicPropertiesService->getPublicStatus($grund);
+      $properties['area_m2'] = intval($grund->getAreal());
+      // @TODO which fields to map for prices?
+      $properties['minimum_price'] = $this->publicPropertiesService->getPublicMinPris($grund);
+      $properties['sale_price']    = $this->publicPropertiesService->getPublicPris($grund);
+      $properties['pdf_link']      = $grund->getPdfLink();
+
+      // Needed in the frontend/weblayer. If true popup will be enabled for the feature.
+      $properties['markers'] = true;
```

Comment:
Alignment of `=` is a bit off here.

I would've written this as

```php
$properties = [
  'id' => $grund->getId(),
  'address' => trim($grund->getVej().' '.$grund->getHusnummer().$grund->getBogstav()),
  …
];
```

as I find it both easier to read and write, but that my personal taste.